python/simple/v1/.env.example

```diff
@@ -0,0 +1,2 @@
+API_KEY=ABCD
```

Comment:
Rename to be more explicit and match other examples https://github.com/amplitude/ampli-examples/blob/main/node/typescript/.env.example